### PR TITLE
Disable emoji parsing in Slack

### PR DESCRIPTION
**Pull request by pierreprinetti**
Avoid polluting the automated Git response with emojis

### CODE REVIEW

**Comment by mandre** [comment]
Oh no! 💥 💔 💥
1 file would be reformatted, 3 files would be left unchanged.

File: src/merge_bot/merge_bot.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def message_slack(webhook_url, msg):
     if webhook_url is None:
         return
     # Cut long messages to 500 chars
-    requests.post(webhook_url, json={"text": msg[:500]})
+    requests.post(webhook_url, json={"blocks":[{"type":"section","text":{"type":"plain_text","text":msg[:500],"emoji":False}}]})
```
Comment (mandre) [inline src/merge_bot/merge_bot.py]: The linter [doesn't like your python style](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/shiftstack_merge-bot/18/pull-ci-shiftstack-merge-bot-main-lint/1716454553118838784#1:build-log.txt%3A69):
```
-    requests.post(webhook_url, json={"blocks":[{"type":"section","text":{"type":"plain_text","text":msg[:500],"emoji":False}}]})
+    requests.post(
+        webhook_url,
+        json={
+            "blocks": [
+                {
+                    "type": "section",
+                    "text": {"type": "plain_text", "text": msg[:500], "emoji": False},
+                }
+            ]
+        },
+    ) 
```

**Comment by pierreprinetti** [comment]
Oh no! 💥 💔 💥

**Review by mandre — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**